docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -0,0 +1,64 @@
+= Parameters
+
+The parent key for all of the following parameters is `metallb`.
+
+
+== `namespace`
+
+[horizontal]
+type:: string
+default:: `syn-metallb`
+
+
+
+== `speaker.secretkey`
+
+[horizontal]
+type:: string
+default:: Vault reference
+
+A binary secret key encoded base64.
+It can be generated using the command `openssl rand -base64 128`.
+
+== `addresses`
+
+[horizontal]
+type:: list
+default:: null
+
+A list of IPs (for example `10.128.1.200/32`) used as transferable virtual IPs.
+
+
+== `config`
+
+[horizontal]
+type:: dictionary
+default:: null
+
+A custom config can be provided.
+This overrides the automatic config generation.
+
+NOTE: The passed configuration is not going to be validated by the component.
+
+For example this allows to specify multiple address pools:
+```
+    config:
+      address-pools:
+        - name: pool1
+          protocol: layer2
+          addresses:
+            - 10.128.1.200/32
+        - name: pool2
+          protocol: layer2
+          addresses:
+            - 10.128.2.200/32
+````
+
+
+== `charts.metallb`
+
+[horizontal]
+type:: git reference
+default:: `3b436a5ec87ffa0772d3d18a27a90dd7c717581f`'
+
+A git reference like a tag, commit, branch etc.
```

Comment:
may make sense to use Git's vocabulary here. This type of reference is usually called a "tree-ish", see https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddeftree-ishatree-ishalsotreeish

Reply:
Was invalid anyway, because we use now the type helm and no longer git. Updated.